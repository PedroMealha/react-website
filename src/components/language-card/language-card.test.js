import { shallow } from "enzyme";
import LanguageCard from "./language-card.component";

describe('LanguageCard component', () => {
	it('expect to render LanguageCard component', () => {
		expect(shallow(<LanguageCard />)).toMatchSnapshot();
	})
})