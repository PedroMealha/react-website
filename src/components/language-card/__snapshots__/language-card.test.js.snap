// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageCard component expect to render LanguageCard component 1`] = `
<div
  className="language-card"
>
  <svg
    viewBox="0 0 200 200"
  >
    <circle
      cx="100"
      cy="100"
      fill="none"
      r="90"
      strokeDashoffset={NaN}
      strokeMiterlimit="10"
      strokeWidth="12"
      transform="rotate(-90)"
    />
    <text
      fontSize="50"
      textAnchor="middle"
      x="100"
      y="115"
    >
      %
    </text>
  </svg>
  <div />
</div>
`;
